app/src/app/user/profile/profile.ts

```diff
@@ -101,6 +101,10 @@ namespace app.user.profile {
 
         }
 
+        public genderOptions():common.models.IGenderOption[] {
```

Comment:
It is more efficient to have this as an injectable from a resolve , or an initialised variable in the constructor; as otherwise the the view will have to re-call this function every time the $digest loop fires.

Basically, views should avoid having function calls as their properties, and use properties on the controller instead.